webrender/examples/animation.rs

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+include!("boilerplate.rs");
```

Comment:
Yup, agreed. This is really great to have the common code separated, but it'd be good to avoid the include!() macro.